vms/platformvm/txs/codec.go

```diff
@@ -97,3 +101,13 @@ func RegisterUnsignedTxsTypes(targetCodec linearcodec.Codec) error {
 	)
 	return errs.Err
 }
+
+func RegisterDUnsignedTxsTypes(targetCodec linearcodec.Codec) error {
+	errs := wrappers.Errs{}
+
+	errs.Add(
+		targetCodec.RegisterType(&TransferSubnetOwnershipTx{}),
+	)
+
+	return errs.Err
```

Comment:
I know we'll probably be adding more txs here... But I think we can make this more complex as needed then
```suggestion
	return targetCodec.RegisterType(&TransferSubnetOwnershipTx{})
```